tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts_full.py

```diff
@@ -225,11 +225,13 @@ def test_edit_after_add(wandb_init):
 
 def test_uploaded_artifacts_are_unstaged(wandb_init, tmp_path, monkeypatch):
     # Use a separate staging directory for the duration of this test.
-    monkeypatch.setenv("WANDB_DATA_DIR", str(tmp_path))
-    staging_dir = Path(wandb_sdk.internal.artifacts.get_staging_dir())
+    monkeypatch.setenv("WANDB_DATA_DIR", str(tmp_path / "staging"))
+    staging_dir = Path(wandb_sdk.internal.staging.get_staging_dir())
 
     def dir_size():
-        return sum(f.stat().st_size for f in staging_dir.rglob("*") if f.is_file())
+        files = [f for f in staging_dir.rglob("*") if f.is_file()]
+        staging_files = [f for f in files if not f.name.startswith(".")]
```

Comment:
Not sure why you added this, can you explain?

Reply:
I'm not actually sure why, but `.netrc` files get created in the temp directory, so instead of finding 4096 bytes it found >1MB. Obviously not always, because this tests was working before.